docs/client/nakama/relay.mdx

```diff
@@ -148,3 +148,59 @@ catch (ApiResponseException ex)
     Debug.LogFormat("Error: {0}", ex.Message);
 }
 ```
+
+### Viewing Receipts and Events
+
+Game clients can view [events](/cardinal/game/system/event) emitted in systems as well as the results of iterating over [messages](/cardinal/game/message#iterating-over-messages) by listening for notifications from Nakama.
+
+#### Example
+
+The code to receive notifications will be different depending on the game engine/client you are using. Learn more about how to receive notifications for your game client at [Nakama docs](https://heroiclabs.com/docs/nakama/concepts/notifications/#receive-notifications).
+
+Here are structs that can be used to deserialize notification information. Note, the Receipt.Result field is an arbitrary object that will depend on the exact messages you've defined in your game.
+
+```csharp Unity/C# Example
+using System;
+using System.Text.Json.Serialization;
+
+public class Event
+{
+    [JsonPropertyName("message")]
+    public string Message { get; set; }
+}
+
+public class Receipt
+{
+    [JsonPropertyName("txHash")]
+    public string TxHash { get; set; }
+
+    [JsonPropertyName("errors")]
+    public List<string> Errors { get; set; }
+
+    // The exact structure of this Result field will depend on the message you defined in your game.
+    [JsonPropertyName("result")]
+    public Dictionary<string, object> Result { get; set; }
+}
+```
+
+This code will listen for Nakama notifications and parse the notifications into receipts and events.
+
+```csharp Unity/C# Example
+var client = new Nakama.Client("http", "127.0.0.1", 7350, "defaultkey");
+var socket = client.NewSocket();
+
+bool appearOnline = true;
+int connectionTimeout = 30;
+await socket.ConnectAsync(Session, appearOnline, connectionTimeout);
+
+socket.ReceivedNotification += notification =>
+{
+    if (notification.Subject == "receipt") {
+        var receipt = JsonSerializer.Deserialize<Receipt>(n.Content)
+        // The exact structure of receipt.Result will be game specific
+    } else if (notification.Subject == "event") {
+        var event = JsonSerializer.Deserialize<Event>(n.Content)
+        // The exact content and structure of event.Message will be game specific
+    }
+};
+```
```

Comment:
JsonSerializer.Deserialize<Receipt>(...) is also not available for us.  I wish it were.
NewtonSoft has a popular JSON library for C# but unfortunately it's not compatible with AOT compilation.
So we have this instead:
var responseTxHash = JsonUtility.FromJson<TxHash>(response.Result.Payload);